AsyncDisplayKit/ASTableView.h

```diff
@@ -457,6 +462,21 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)tableView:(ASTableView *)tableView didEndDisplayingNodeForRowAtIndexPath:(NSIndexPath *)indexPath ASDISPLAYNODE_DEPRECATED;
 
+/**
+ * Informs the delegate that the table view will add the node
+ * at the given index path to the view hierarchy.
+ *
+ * @param tableView The sender.
+ * @param indexPath The index path of the row that will be displayed.
+ *
+ * @warning AsyncDisplayKit processes table view edits asynchronously. The index path
+ *   passed into this method may not correspond to the same item in your data source
+ *   if your data source has been updated since the last edit was processed.
+ *
+ * This method is deprecated. Use @c tableView:willDisplayNode:forRowAtIndexPath: instead.
```

Comment:
That means "print the next word in monospaced font" http://blog.dadabeatnik.com/2013/09/25/comment-docs-in-xcode-5/